### PR TITLE
Add VST Plugin

### PR DESCRIPTION
**Pull request by DDRBoxman**
I'd love it if I could also get a code review over at https://github.com/DDRBoxman/obs-vst

### CODE REVIEW

**Comment by WizardCM** [comment]
Yes! I'm excited for this one. :D

File: .gitmodules [hunk 1]
```diff
@@ -11,3 +11,6 @@
 [submodule "plugins/obs-browser"]
 	path = plugins/obs-browser
 	url = https://github.com/kc5nra/obs-browser.git
+[submodule "plugins/obs-vst"]
+	path = plugins/obs-vst
+	url = git@github.com:DDRBoxman/obs-vst.git
```
Comment (reboot) [inline .gitmodules]: You should probably replace this with an http URL like all the other submodules.

**Comment by ONSBalder** [comment]
Can one of the admins verify this patch?

**Comment by DDRBoxman** [comment]
Updated the submodule reference to match a bunch of changes that @c3r1c3 made

**Comment by jp9000** [comment]
Fixed it so that it detects whether the submodule is present to prevent it from breaking builds if submodules aren't properly fetched.

**Comment by kszere** [comment]
When VST for Linux? :)

**Pull request closed without merging**